code/game/objects/structures/flora.dm

```diff
@@ -253,11 +253,11 @@
 	. = ..()
 	icon_state = "fullgrass_[rand(1, 3)]"
 
-
+// Kirby plants
 /obj/item/kirbyplants
 	name = "potted plant"
 	icon = 'icons/obj/flora/plants.dmi'
-	icon_state = "plant-1"
+	icon_state = "random_plant"
 	anchored = FALSE
 	layer = ABOVE_MOB_LAYER
 	w_class = WEIGHT_CLASS_HUGE
```

Comment:
using `icon_state` to inform special behaviour is bad practice, make a `/obj/item/kirbyplants/random` subtype please

Reply:
In that case, it would be necessary to replace every plant on all the maps.....

Reply:
a future TODO I guess then